src/modules/Builder/features/SaveAndPublish/SaveAndPublish.hooks.ts

```diff
@@ -325,8 +325,8 @@ export const useSaveAndPublishSetup = (hasPrompt: boolean) => {
       }
 
       if (appletId && ownerId) {
-        navigate(getBuilderAppletUrl(appletId));
         await dispatch(getAppletWithItems({ ownerId, appletId }));
+        navigate(getBuilderAppletUrl(appletId));
```

Comment:
As I remember, there was an issue relating to the order in these rows. @muhurov could you please provide more details?